ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+I'm submitting a ... (check one with "x")
+- [ ] bug report
+- [ ] feature request
+
+***For bug reports, please include:***
```

Comment:
Comment this

Reply:
Also, instruct the user to delete everything below here if it's not a bug report